### PR TITLE
Create laravel.yml

### PR DESCRIPTION
**Pull request by vIremAydin**
add action

- - -
This change is [Reviewable](https://reviewable.io/reviews/Ece-Kahraman/Eventium/2)

### CODE REVIEW

**Comment by Ece-Kahraman** [comment]
no help 🗡️

**Review by vIremAydin — COMMENTED** [review]
gfbfgbf

> *[Reviewable](https://reviewable.io/reviews/Ece-Kahraman/Eventium/2#-:-NjsVJzWCYyzVtUY1Rlo:b43we3r)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @Ece-Kahraman)

*[a discussion](https://reviewable.io/reviews/Ece-Kahraman/Eventium/2#-NjsVLYzCuhu2MXOMp1H:-NjsVLYzCuhu2MXOMp1I:b-oxcs9y) (no related file):*
jhmbhj

___

**Comment by AlperMumcular** [comment]
hello

File: .github/workflows/laravel.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    branches: [ "main" ]
+
+jobs:
+  laravel-tests:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: shivammathur/setup-php@15c43e89cdef867065b0213be354c2841860869e
+      with:
```
Comment (vIremAydin) [inline .github/workflows/laravel.yml:16]: sdacscscsd

Comment (vIremAydin) [reply .github/workflows/laravel.yml:16]: dsacdscds

Comment (vIremAydin) [reply .github/workflows/laravel.yml:16]: sefsfsdsdaf

**Comment by vIremAydin** [comment]
vxczvcxvcxv

**Comment by vIremAydin** [comment]
> hello

wefwdesfe

File: .github/workflows/laravel.yml (hunk 1 above)
Comment (vIremAydin) [reply .github/workflows/laravel.yml:16]: nomnomnom

File: .github/workflows/laravel.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+name: Laravel
+
+on:
```
Comment (Ece-Kahraman) [inline .github/workflows/laravel.yml:3]: THOUGHT (IF-MINOR): this is a thought

File: .github/workflows/laravel.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  push:
+    branches: [ "main" ]
+  pull_request:
```
Comment (Ece-Kahraman) [inline .github/workflows/laravel.yml:6]: ISSUE (BLOCKING): this is an issue

**Review by Ece-Kahraman — COMMENTED** [review]
hello

File: .github/workflows/laravel.yml (hunk 2 above)
Comment (Ece-Kahraman) [inline .github/workflows/laravel.yml:3]: THOUGHT (IF-MINOR): this is a thought

File: .github/workflows/laravel.yml (hunk 3 above)
Comment (Ece-Kahraman) [inline .github/workflows/laravel.yml:6]: ISSUE (BLOCKING): this is an issue

**Review by Ece-Kahraman — COMMENTED** [review]
hello

File: .github/workflows/laravel.yml (hunk 2 above)
Comment (Ece-Kahraman) [inline .github/workflows/laravel.yml:3]: THOUGHT (IF-MINOR): this is a thought

File: .github/workflows/laravel.yml (hunk 3 above)
Comment (Ece-Kahraman) [inline .github/workflows/laravel.yml:6]: ISSUE (BLOCKING): this is an issue

**Review by Ece-Kahraman — COMMENTED** [review]
hello 3

**Review by Ece-Kahraman — COMMENTED** [review]
hello 4

File: .github/workflows/laravel.yml (hunk 2 above)
Comment (Ece-Kahraman) [inline .github/workflows/laravel.yml:3]: THOUGHT (IF-MINOR): this is a thought

File: .github/workflows/laravel.yml (hunk 3 above)
Comment (Ece-Kahraman) [inline .github/workflows/laravel.yml:6]: ISSUE (BLOCKING): this is an issue

**Review by Ece-Kahraman — COMMENTED** [review]
hello 4

File: .github/workflows/laravel.yml (hunk 3 above)
Comment (Ece-Kahraman) [inline .github/workflows/laravel.yml:6]: ISSUE (BLOCKING): this is an issue

**Review by Ece-Kahraman — COMMENTED** [review]
hello 5

File: .github/workflows/laravel.yml (hunk 2 above)
Comment (Ece-Kahraman) [inline .github/workflows/laravel.yml:3]: THOUGHT (IF-MINOR): this is a thought

File: .github/workflows/laravel.yml (hunk 3 above)
Comment (Ece-Kahraman) [inline .github/workflows/laravel.yml:6]: ISSUE (BLOCKING): this is an issue

**Review by Ece-Kahraman — COMMENTED** [review]
hello 6

File: .github/workflows/laravel.yml (hunk 2 above)
Comment (Ece-Kahraman) [inline .github/workflows/laravel.yml:3]: THOUGHT (IF-MINOR): this is a thought

File: .github/workflows/laravel.yml (hunk 3 above)
Comment (Ece-Kahraman) [inline .github/workflows/laravel.yml:6]: ISSUE (BLOCKING): this is an issue

**Review by Ece-Kahraman — COMMENTED** [review]
hello 7

File: .github/workflows/laravel.yml (hunk 2 above)
Comment (Ece-Kahraman) [inline .github/workflows/laravel.yml:3]: THOUGHT (IF-MINOR): this is a thought

File: .github/workflows/laravel.yml (hunk 3 above)
Comment (Ece-Kahraman) [inline .github/workflows/laravel.yml:6]: ISSUE (BLOCKING): this is an issue

**Review by Ece-Kahraman — COMMENTED** [review]
hello 8

**Comment by Ece-Kahraman** [comment]
**ACTIVE**: <p>comment editleme</p>

**Pull request merged**